template/page-ambassadors.html.jinja

```diff
@@ -287,136 +277,155 @@
           <div class="carousel-item active">
             <div class="p-0">
                 <div class="mb-4 p-lg-3">
-                    <img src="./assets/img/ambassadors/quote1-small.svg" class="h-100 d-lg-none mb-4">
-                    <img src="./assets/img/ambassadors/quote1.svg" class="h-100 d-none d-lg-block">
+                    <img alt="quote-1-small" src="./assets/img/ambassadors/quote1-small.svg" class="h-100 d-lg-none mb-4">
+                    <img alt="quote-1-big" src="./assets/img/ambassadors/quote1.svg" class="h-100 d-none d-lg-block">
                     <div class="p-0 col-lg-7 mx-lg-auto">
-                        <p class="p-lg-3 mb-2"><strong>Titose C.</strong><br >
-                            University of Cape Town,<br >
-                            Spring 2022 XRPL Campus Ambassador</p>
+                        <p class="p-lg-3 mb-2"><strong>Derrick N.</strong><br >
+                            Toronto Metropolitan University<br >
+                            Spring 2023 XRPL Campus Ambassador</p>
                     </div>
                 </div>
 
             </div>
           </div>
-          <div class="carousel-item">
+          <div class="carousel-item mb-20">
             <div class="p-0">
                 <div class="mb-4 p-lg-3">
-                    <img src="./assets/img/ambassadors/quote2-small.svg" class="h-100 d-lg-none  mb-4">
-                    <img src="./assets/img/ambassadors/quote2.svg" class="h-100 d-none d-lg-block">
+                    <img alt="quote-2-small" src="./assets/img/ambassadors/quote2-small.svg" class="h-150 d-lg-none  mb-4">
```

Comment:
"quote-2-small" is not appropriate alt text.

W3C recommends using CSS instead of an image, but failing that, the alt text should be the same text as in the image:

> [Images of text](https://www.w3.org/WAI/tutorials/images/textual/): Readable text is sometimes presented within an image. If the image is not a logo, avoid text in images. However, if images of text are used, the text alternative should contain the same words as in the image.

```suggestion
                    <img alt="The XRPL Campus Ambassador program really helped broaden my view of the blockchain industry with their learning resource and virtual community. Being an ambassador allowed me to meet industry professionals and likeminded peers which have given me invaluable experiences and insights." src="./assets/img/ambassadors/quote2-small.svg" class="h-150 d-lg-none  mb-4">
```

That said, I think it _would_ also be possible to use CSS to re-create this effect, e.g. using `background-clip` ([example tutorial](https://www.coding-dude.com/wp/css/applying-an-image-texture-on-text-with-html-and-css/)). You could probably also improve the contrast in light mode while you're at it.